tailwind.config.js

```diff
@@ -4,39 +4,40 @@ const colors = require('tailwindcss/colors');
 module.exports = {
   content: ['src/**/*.{js,ts,jsx,tsx}'],
   theme: {
-    colors: {
-      // default
-      black: colors.black,
-      transparent: colors.transparent,
-      blue: colors.blue,
-      red: colors.red,
-      yellow: colors.amber,
-      pear: '#C8D72B',
-      teal: colors.teal,
-      // custom
-      green: '#7ED8C2',
-      red: '#F17256',
-      orange: {
-        DEFAULT: '#E7B65D',
-        700: '#E7B65D',
-        800: '#DFA849',
-        900: '#EEAF43',
+    extend: {
+      colors: {
+        // default
+        black: colors.black,
```

Comment:
Extend doesn’t mean we do not have to add default values?

Reply:
Right. Removed.